cmd/account/mgmt/account-unassign.go

```diff
@@ -0,0 +1,405 @@
+		if err != nil {
+			return err
+		}
+	}
+
+	fmt.Printf("Are you sure you want to unassign the accounts? [y/n] ")
+	reader := bufio.NewReader(os.Stdin)
+
+	response, err := reader.ReadString('\n')
+	if err != nil {
+		return err
+	}
+
+	if response == "y" || response == "Y" {
+
+		// loop through accounts list and untag and move them back into root OU
+		for _, id := range accountIdList {
+
+			err = o.untagAccount(id)
+			if err != nil {
+				return err
+			}
+
+			err = o.moveAccount(id, rootID, destinationOU)
+			if err != nil {
+				return err
+			}
+		}
+
+		if accountUsername != "" {
+			// Delete login profile
+			err := o.deleteLoginProfile(accountUsername)
+			if err != nil {
+				return err
+			}
+			// Delete access keys
+			err = o.deleteAccessKeys(accountUsername)
+			if err != nil {
+				return err
+			}
+			// Delete signing certificates
+			err = o.deleteSigningCert(accountUsername)
+			if err != nil {
+				return err
+			}
+			// Delete policies
+			err = o.deletePolicies(accountUsername)
+			if err != nil {
+				return err
+			}
+			// Delete attached policies
+			err = o.deleteAttachedPolicies(accountUsername)
+			if err != nil {
+				return err
+			}
+			// Delete groups
+			err = o.deleteGroups(accountUsername)
+			if err != nil {
+				return err
+			}
+			// Delete user
+			err = o.deleteUser(accountUsername)
+			if err != nil {
+				return err
+			}
+		}
+
+	}
+	return nil
+}
+
+var ErrHiveNameProvided error = fmt.Errorf("non-ccs account provided, only developers account accepted")
+var ErrAccountPartiallyTagged error = fmt.Errorf("account is only partially tagged")
+
+func (o *accountUnassignOptions) checkForHiveNameTag(id string) error {
+
+	inputListTags := &organizations.ListTagsForResourceInput{
+		ResourceId: &id,
+	}
+	tags, err := o.awsClient.ListTagsForResource(inputListTags)
+	if err != nil {
+		return err
+	}
+
+	if len(tags.Tags) == 0 {
+		return ErrNoTagsOnAccount
+	}
+
+	for _, t := range tags.Tags {
+		if *t.Key == "owner" && strings.HasPrefix(*t.Value, "hive") {
+			return ErrHiveNameProvided
+		}
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) untagAccount(id string) error {
+	inputUntag := &organizations.UntagResourceInput{
+		ResourceId: &id,
+		TagKeys: []*string{
+			aws.String("owner"),
+			aws.String("claimed"),
+		},
+	}
+	_, err := o.awsClient.UntagResource(inputUntag)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) moveAccount(id string, rootID string, destinationOU string) error {
+	inputMove := &organizations.MoveAccountInput{
+		AccountId:           aws.String(id),
+		DestinationParentId: aws.String(rootID),
+		SourceParentId:      aws.String(destinationOU),
+	}
+
+	_, err := o.awsClient.MoveAccount(inputMove)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+var ErrNoAccountsForUser error = fmt.Errorf("user has no aws accounts")
+
+func (o *accountUnassignOptions) listAccountsFromUser(user string) ([]string, error) {
```

Comment:
Is this something that could be exported and used in the assign accounts command?